### ISSUE TITLE
Replace deprecated links 

### ISSUE THREAD

**Issue by fm-sys** [maks/viper-edit#11]
Many links inside the app aren't up to date anymore. 

Question @maks: do you accept PR's and are you still actively maintaining the fork (do Playstore releases, etc)? If so, I would happily contribute to the repo... 

![Screenshot_20210728_124313](https://user-images.githubusercontent.com/64581222/127310752-35fe3794-8cbb-441b-897f-f60452e648f8.jpg)

**Comment by maks** [comment]
Yes I'm am still maintaining this fork and doing releases on Play. And yes am happy to receive PRs.

**Comment by maks** [comment]
And I'm not a native German speaker so my German is quite sketchy, so happy to receive translation updates.

**Comment by fm-sys** [comment]
> so happy to receive translation updates.

What about setting up [GitLocalize](https://gitlocalize.com/repo/new) for this repository? That would probably make transactions a lot easier... (It's free BTW 😉) 

An alternative tool would be Crowdin (free as well for open source): 
https://crowdin.com/page/open-source-project-setup-request

I already have good experiences with such tools and there are quickly people who gladly help with translations if you add a small link to the tool somewhere not too hidden into the app...
